package.json

```diff
@@ -7,18 +7,24 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start-mock-api": "NODE_ENV=development node lib/start-mock-api.mjs"
   },
   "dependencies": {
     "next": "12.1.0",
     "react": "17.0.2",
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "date-fns": "^2.28.0",
+    "dotenv": "^16.0.0",
+    "dotenv-expand": "^8.0.3",
     "eslint": "8.10.0",
     "eslint-config-next": "12.1.0",
+    "fastify": "^3.27.4",
     "husky": "^7.0.4",
     "lint-staged": "^12.3.7",
+    "node-fetch": "^3.2.3",
```

Comment:
Many of these dependencies wouldn't normally be dev-only, but they are for this project because they are only used in development when the mock API server gets brought up.